Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Hotfix for BTS report #863527:
In Portuguese and Brazilian documents don't break on a " as last character
in a table title.
--- a/latex/style/dbk_locale.sty
+++ b/latex/style/dbk_locale.sty
@@ -286,6 +286,9 @@
 \babelsetup{nn}{\AtBeginDocument{\shorthandoff{"}}}
 \babelsetup{nb}{\AtBeginDocument{\shorthandoff{"}}}
 
+\babelsetup{pt}{\AtBeginDocument{\shorthandoff{"}}}
+\babelsetup{pt_br}{\AtBeginDocument{\shorthandoff{"}}}
+
 %% Apply default locale setup
 \setuplocale{en}
 
